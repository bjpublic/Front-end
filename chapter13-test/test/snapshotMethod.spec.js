function getHelloButtonHtml() {
  return "<button>Hello</button>";
}

describe("snapshot 메서드 테스트", () => {
  it("toMatchInlineSnapshot()은 파일 내부에 스냅샷을 기록한다.", () => {
    expect(getHelloButtonHtml()).toMatchInlineSnapshot(`
      <button>
        Hello
      </button>
    `);
  });

  it("toMatchSnapshot()은 파일 외부에 스냅샷을 기록한다.", () => {
    expect(getHelloButtonHtml()).toMatchSnapshot();
  });
});
