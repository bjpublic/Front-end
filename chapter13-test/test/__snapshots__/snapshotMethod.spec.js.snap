// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 메서드 테스트 toMatchSnapshot()은 파일 외부에 스냅샷을 기록한다. 1`] = `
<button>
  Hello
</button>
`;
